docs/developer-guide/maven-indexer.md

```diff
@@ -55,6 +55,14 @@ The following file types are not indexable:
 
 The Maven Indexer is used for integration with IDE-s.
 
+## How Does The Maven Indexer Work In Strongbox ?
+
+Every maven repository with indexing enabled serves the packed Maven Index. Strongbox stores the information of uploaded artifacts 
+(in hosted repositories) in the OrientDB database. This Information is used to create the hosted repository Maven Index.
+
+Proxy repository TBD ....
+used to
```

Comment:
I assume this is still work in progress...?

Reply:
yes, see label